pkg/ocicni/util_freebsd.go

```diff
@@ -10,23 +10,24 @@ import (
 	"strings"
 )
 
-var defaultJexecCommandName = "jexec"
-
 type nsManager struct {
-	jexecPath string
 }
 
 func (nsm *nsManager) init() error {
 	var err error
-	nsm.jexecPath, err = exec.LookPath(defaultJexecCommandName)
 	return err
 }
 
 func getContainerDetails(nsm *nsManager, netnsJailName, interfaceName, addrType string) (*net.IPNet, *net.HardwareAddr, error) {
 	// Try to retrieve ip inside container network namespace
+	if addrType == "-4" {
+		addrType = "inet"
+	} else if addrType == "-6" {
+		addrType = "inet6"
+	}
```

Comment:
Just a small nit. This would be fine with if-else only, per:

```suggestion
	if addrType == "-4" {
		addrType = "inet"
	} else {
		addrType = "inet6"
	}
```

Given how this function is invoked. Also curious: passing "-4", and such, does not work any more?

Reply:
As far as I can tell, it never worked. When I wrote this originally, I tested with ```ocicnitool``` but did not realise that the code which would ```getContainerDetails``` was getting its values from CNI instead.

Reply:
Makes sense. Thank you! :+1: